### DISCUSSION TITLE
这次更新遇到了新问题（1.10.2）

### DISCUSSION THREAD

**Discussion by GitHubmnxz** [messense/aliyundrive-webdav#645] [Q&A]
Collected errors:

pkg_hash_check_unresolved: cannot find dependency luci-lua-runtime for luci-app-aliyundrive-webdav
pkg_hash_fetch_best_installation_candidate: Packages for luci-app-aliyundrive-webdav found, but incompatible with the architectures configured
opkg_install_cmd: Cannot install package luci-app-aliyundrive-webdav.

**Comment by GitHubmnxz** [comment]
降级到1.10.1正常使用

**Comment by messense** [comment]
1.10.3 应该修复了。
